mindmeld/active_learning/alp.py

```diff
@@ -44,6 +45,7 @@ def __init__(  # pylint: disable=R0913
         batch_size: int,
         tuning_strategies: list,
         tuning_level: str,
+        entity_tuning: bool,
```

Comment:
Just to clarify - we only have a single strategy for entities (irrespective of the model) for now?

Reply:
No, we still have three strategies for entities. LCS, Entropy and Margin.